### ISSUE TITLE
Only enable archiving once the 'heartbeat' is done

### ISSUE THREAD

**Issue by grahams** [grahams/trackupdate#14]
Right now the XML generator will ignore tracks in the blacklist, but this just shifts all of the track times inappropriately.  It would be a better idea to remove that code, but instead have the script enable archiving once the lead-in 'heartbeat' is completed..  This would require a config option to identify the heartbeat.

**Comment by grahams** [comment]
the XML plugin as well as the whole notion of enabling archiving is no longer relevant
